hnn_core/gui/gui.py

```diff
@@ -398,7 +421,102 @@ def compose(self):
                                     self.widget_tstop, self._load_info,
                                     self.layout)
 
-        return hnn_gui
+        if not return_layout:
+            return
+        else:
+            return self.app_layout
+
+    def show(self):
+        display(self.app_layout)
+
+    def take_screenshot(self, width=None, height=None, filename=None,
+                        render=True):
+        """Embed the widget to HTML and render it through iframe.
+        Parameters
+        ----------
+        width : Optional[int]
```

Comment:
```suggestion
        width : int
```

to be consistent with rest of repo